src/ca/bc/gov/nrids/polaris/JenkinsUtil.groovy

```diff
@@ -8,6 +8,12 @@ class JenkinsUtil implements Serializable {
     development: "dev"
   ]
 
+  private final static def envShortToLong = [
+    prod: "production",
+    test: "test",
+    dev: "development"
+  ]
```

Comment:
Duplication. Invert the existing map. See: https://chatgpt.com/share/390de9eb-c5f1-469d-8432-7179b5d2a529